storage/ndb/src/kernel/blocks/dbspj/DbspjMain.cpp

```diff
@@ -26,37 +26,37 @@
 #include "Dbspj.hpp"
 
 #include <ndb_version.h>
+#include <AttributeDescriptor.hpp>
```

Comment:
Is this just clang-formatting?
I doubt we ever touched NDB code

Reply:
There was a conflict in this file for some reason... and then clang-format managed to reformat everything. I'll ifx it.